IDS/Log/Email.php

```diff
@@ -160,7 +167,6 @@ protected function __construct($config)
         } elseif (is_array($config)) {
             $this->recipients[]      = $config['recipients'];
             $this->subject           = $config['subject'];
-            $this->additionalHeaders = $config['header'];
```

Comment:
`additionalHeaders` was not being used anywhere, so rather than declare the property, I removed this line.